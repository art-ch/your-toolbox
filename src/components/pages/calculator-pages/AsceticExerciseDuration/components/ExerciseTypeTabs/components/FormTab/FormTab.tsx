import { TabsContent } from '@/components/ui/tabs';
import React from 'react';

export type FormTabProps = {
  children: React.ReactNode;
  value: string;
};

export const FormTab = ({ children, value }: FormTabProps) => (
  <TabsContent value={value}>
    <section className="flex flex-col gap-y-8 py-2 sm:py-4 md:py-6">
      {children}
    </section>
  </TabsContent>
);
