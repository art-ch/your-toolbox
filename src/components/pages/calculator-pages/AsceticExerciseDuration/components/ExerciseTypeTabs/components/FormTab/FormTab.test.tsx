import { render } from '@testing-library/react';
import { Tabs } from '@/components/ui/tabs';
import { FormTab } from './FormTab';

describe('FormTab', () => {
  it('should render', () => {
    const { container } = render(
      <Tabs value="test">
        <FormTab value="test">
          <div>Child</div>
        </FormTab>
      </Tabs>
    );

    expect(container).toMatchSnapshot();
  });
});
