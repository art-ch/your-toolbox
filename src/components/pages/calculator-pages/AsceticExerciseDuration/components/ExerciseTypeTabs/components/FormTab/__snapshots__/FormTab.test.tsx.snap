// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormTab should render 1`] = `
<div>
  <div
    class="flex flex-col gap-2"
    data-orientation="horizontal"
    data-slot="tabs"
    dir="ltr"
  >
    <div
      aria-labelledby="radix-«r0»-trigger-test"
      class="flex-1 outline-none"
      data-orientation="horizontal"
      data-slot="tabs-content"
      data-state="active"
      id="radix-«r0»-content-test"
      role="tabpanel"
      style="animation-duration: 0s;"
      tabindex="0"
    >
      <section
        class="flex flex-col gap-y-8 py-2 sm:py-4 md:py-6"
      >
        <div>
          Child
        </div>
      </section>
    </div>
  </div>
</div>
`;
